src/Illuminate/Encryption/Encrypter.php

```diff
@@ -32,16 +22,21 @@ class Encrypter implements EncrypterContract
      *
      * @throws \RuntimeException
      */
-    public function __construct($key, $cipher = 'AES-128-CBC')
+    public function __construct($key = null, $cipher = 'AES-128-CBC')
```

Comment:
Please update the phpdoc to `string|null`

Reply:
Sorry for the mess, I think I fixed this.